### ISSUE TITLE
Expose player.isMuted()

### ISSUE THREAD

**Issue by mburns-acom** [PierfrancescoSoffritti/android-youtube-player#1117]
# Feature Request
I would like to expose the API method `player.isMuted():Boolean`.
How can this be done?

### Describe your idea

Currently we can set the mute/unmute state on creation, but we would like to know if the user has unmuted the video. It looks like the iFrame api supports

**Comment by PierfrancescoSoffritti** [comment]
Currently we don't have a mechanism to invoke javascript methods that return a value. It should be figured out before we can do this.

**Comment by mburns-acom** [comment]
That's what I figured. If we can invoke a javascript method that returns the value, then we could add this feature. Correct?

**Comment by PierfrancescoSoffritti** [comment]
Correct. There is probably a way to design a nice API using coroutines.
